tools/src/ceeuopt.s

```diff
@@ -0,0 +1,77 @@
+               10,0,10,0,10,0,10,0,10,0,10,0,10),                      X00410000
+               HEAPZONES=(0,ABEND,0,ABEND),                            X00415000
+               INFOMSGFILTER=(OFF,,,,),                                X00420000
+               INQPCOPN=(ON),                                          X00430000
+               INTERRUPT=(OFF),                                        X00440000
+               LIBSTACK=(4K,4K,FREE),                                  X00460000
+               MSGFILE=(SYSOUT,FBA,121,0,NOENQ),                       X00470000
+               MSGQ=(15),                                              X00480000
+               NATLANG=(ENU),                                          X00490000
+               NOAUTOTASK=,                                            X00500000
+               NOTEST=(ALL,*,PROMPT,INSPPREF),                         X00520000
+               NOUSRHDLR=(''),                                         X00530000
+               OCSTATUS=(ON),                                          X00540000
+               PAGEFRAMESIZE=(4K,4K,4K),                               X00541000
+               PC=(OFF),                                               X00550000
+               PLITASKCOUNT=(20),                                      X00560000
+               POSIX=(ON),                                             X00570000
+               PROFILE=(OFF,''),                                       X00580000
+               PRTUNIT=(6),                                            X00590000
+               PUNUNIT=(7),                                            X00600000
+               RDRUNIT=(5),                                            X00610000
+               RECPAD=(OFF),                                           X00620000
+               RPTOPTS=(OFF),                                          X00630000
+               RPTSTG=(OFF),                                           X00640000
+               RTEREUS=(OFF),                                          X00650000
+               SIMVRD=(OFF),                                           X00670000
+               STACK=(128K,128K,ANYWHERE,KEEP,512K,128K),              X00680000
+               STORAGE=(NONE,NONE,NONE,0K),                            X00690000
+               TERMTHDACT=(TRACE,,96),                                 X00700000
+               THREADHEAP=(4K,4K,ANYWHERE,KEEP),                       X00710000
+               THREADSTACK=(OFF,4K,4K,ANYWHERE,KEEP,128K,128K),        X00720000
+               TRACE=(OFF,4K,DUMP,LE=0),                               X00730000
+               TRAP=(ON,SPIE),                                         X00740000
+               UPSI=(00000000),                                        X00750000
+               VCTRSAVE=(OFF),                                         X00760000
+               XPLINK=(OFF),                                           X00780000
```

Comment:
XPLINK is off?

Do we need all of these settings or just the FILETAG one?

Reply:
These settings should be all the defaults - so it's a little verbose but is kind of useful so you see all the options.
Note that this code is built EBCDIC, 31-bit since it needs to work with the web toolkit code that is like this.